chat-server/main.js

```diff
@@ -0,0 +1,53 @@
+
+// Connect to the Socket.io server
+const socket = io('http://localhost:9090');
```

Comment:
Nice use of sockets! In terms of completeness (you don't have to do it), would you say that your socket implementation would also have a deleteMessage function, similar to addMessage?